packages/server/src/fhir/operations/db-configure-indexes.ts

```diff
@@ -83,6 +87,9 @@ type OutputAction = {
 
 export async function dbConfigureIndexesHandler(req: FhirRequest): Promise<FhirResponse> {
   requireSuperAdmin();
+  if (req.headers?.['prefer'] !== 'respond-async') {
```

Comment:
Small nit, do we also want to export `requireAsync` from `super.ts` and use it here like `requireSuperAdmin`?

Reply:
Ah yeah not a big deal then 👍